packages/calypso-e2e/src/lib/components/support-component.ts

```diff
@@ -182,8 +183,30 @@ export class SupportComponent extends BaseContainer {
 
 		await items[ target ].click();
 		await this.page.click( selectors.readMoreButton );
-		const supportArticleComponent = await SupportArticleComponent.Expect( this.page );
-		await supportArticleComponent.articleDisplayed();
+	}
+
+	/**
+	 * Visit the support article from the inline support popover.
+	 *
+	 * @returns {Promise<Page>} Reference to support page.
+	 */
+	async visitArticle(): Promise< Page > {
+		const browserContext = this.page.context();
+		const [ newPage ] = await Promise.all( [
+			browserContext.waitForEvent( 'page' ),
+			this.page.click( selectors.visitArticleButton ),
+		] );
+		await newPage.waitForLoadState( 'domcontentloaded' );
+		return newPage;
```

Comment:
TIL. Nice way of handling the link to a new page. Do you think this should be a default way of handling link clicks to a new page?